cluster_library.c

```diff
@@ -883,6 +891,48 @@ cluster_free(redisCluster *c, int free_ctx)
     if (free_ctx) efree(c);
 }
 
+static zend_long cluster_cache_expiry(void) {
+    zend_long expiry;
+
+    expiry = INI_INT("redis.clusters.slot_cache_expiry");
+    if (expiry <= 0)
+        return 0;
+
+    return time(NULL) + expiry;
```

Comment:
Shouldn't we use monotonic clock here?

Reply:
Good point, probably. I'll need to see what the right way to do that is so it also compiles on Windows.